career_paths/ml_engineer.md

```diff
@@ -46,7 +60,7 @@ One may not need to know all of them, but one technology from each area would ma
 * MLFlow
 * Sagemaker
 
-#### MLOps
+#### Deployment
 * Docker
 * Kubernetes
```

Comment:
Maybe add "KFServing" (part of kubeflow) and TF-serving?

Reply:
hm, sorry, didn't notice it's for ML engineer. Probably it makes sense for both mlops and MLE